client/a8c-for-agencies/components/user-contact-support-modal-form/style.scss

```diff
@@ -54,9 +50,7 @@
 }
 
 .a4a-contact-support-modal-form__title {
-	font-size: 1.5rem;
-	font-weight: 700;
-	line-height: 0.8;
+	@include a4a-font-heading-xl;
```

Comment:
`font-weight` here is 700, and in the `a4a-font-heading-xl` it is 600. Is that ok?